src/seed/seeds.ts

```diff
@@ -24,15 +24,21 @@ async function generateRandomData() {
   };
 }
 
-async function seedDatabase() {
+export async function seedDatabase(maxUsers: number) {
```

Comment:
```suggestion
export async function seedDatabase(maxUsers = 50) {
```

Valor padrão de um argumento. Assim vc não precisa passar parâmetro quando quiser gerar 50.